querybook/server/lib/ai_assistant/base_ai_assistant.py

```diff
@@ -207,6 +207,8 @@ def generate_sql_query(
             )
             if tables:
                 socket.send_tables_for_sql_gen(tables)
+                socket.close()
+                return
```

Comment:
lets just refactor the return in the next one in the same if states, so its more like if tables are not provided, no sql is getting generated

Reply:
maybe a separate function would help with the readability?